draft-ietf-ccwg-rfc5033bis.md

```diff
@@ -265,17 +265,78 @@ fall outside the scope of these guidelines are certainly possible;
+consider the following criteria. These criteria will be evaluated in various
+domains (see {{general-use}} and {{special-cases}}).
 
-: Proposed congestion control mechanisms should include a clear
-explanation of the deviations from {{!RFC2914}}.
+## Single Algorithm Behavior
+
+The following criteria evaluate the proposed algorithm when one or more
+flows using that algorithm share a bottleneck link, with no other algorithms
+operating.
+
+### Protection Against Congestion Collapse
+
+The alternate congestion control mechanism should either stop
+sending when the packet drop rate exceeds some threshold
+{{?RFC3714}}, or should include some notion of "full backoff".  For
+"full backoff", at some point the algorithm would reduce the
+sending rate to one packet per round-trip time and then
+exponentially backoff the time between single packet
+transmissions if congestion persists.  Exactly when either "full
+backoff" or a pause in sending comes into play will be
+algorithm-specific.  However, as discussed in {{!RFC2914}}, this
+requirement is crucial to protect the network in times of extreme
+congestion.
+
+If "full backoff" is used, this bullet does not require that the
+full backoff mechanism must be identical to that of TCP
+{{?RFC2988}}.  As an example, this bullet does not preclude full
+backoff mechanisms that would give flows with different round-
+trip times comparable caapcity during backoff.
+
+### Protection Against Bufferbloat
 
-(1)
-: Impact on existing deployments of TCP {{!RFC9293}}, SCTP {{!RFC9260}},
-DCCP {{!RFC4340}}, and QUIC {{!RFC9000}}.
+The alternate congestion control mechanism should reduce its sending
+rate if the round trip time (RTT) significantly increases. Exactly how
+the algorithm reduces its sending rate is algorithm specific.
 
-: Proposed congestion control mechanisms should be evaluated when
+Bufferbloat {{Bufferbloat}} refers to the building of long queues in
+the network. Many network routers are configured with very large buffers.
+If congestion starts happening, classic TCP congestion control algorithms
+{{!RFC5681}} will continue sending at a high rate until the buffer fills
+up completely and packet losses occur. Every connection going through
+that bottleneck will experience high latency.
+This is particularly bad for highly interactive applications like games,
+but it also affects routine web browsing and video playing.
+
+This problem became apparent in the last decade and was not discussed in
+the Congestion Control Principles published in September 2002 {{!RFC2914}}.
+The classic congestion control algorithm {{!RFC5681}} and the widely deployed
+Cubic algorithm {{?RFC9438}} do not address it, but newly designed congestion
+control algorithms have the opportunity to improve the state of the art.
+
+### Fairness within the Alternate Congestion Control Algorithm.
+
+In environments with multiple competing flows all using the same
+alternate congestion control algorithm, the proposal should
```

Comment:
Unsure here : This is really great guidance. CCs are designed and used in many deployment environments. I think we need to remove /In environments with/ and add /When/ or something. We will probably also need to work out more detail here later.